vehicles/src/common/enum/application-status.enum.ts

```diff
@@ -12,4 +12,26 @@ export enum ApplicationStatus {
   SUPERSEDED = 'SUPERSEDED',
   REVOKED = 'REVOKED',
   VOIDED = 'VOIDED',
+  DELETED = 'DELETED',
 }
+
+/**
+ * Apllication statuses to be considered for Application In Progress (AIP) from IDIR User POV.
+ */
+export const idirUserAIPStatus: ApplicationStatus[] = [
```

Comment:
use ReadOnlyArray to make it immutable.
Refer: https://www.educative.io/courses/learn-typescript-complete-course/mutable-and-immutable-arrays#Immutable-arrays

Reply:
Done.